docs/networks/network-architecture/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Network Architecture
+sidebar_position: 1
+---
+
```

Comment:
Add a Title of "Flow's Network Architecture"

Reply:
made the change - but the title does not show up. I _think_ that's because I have a header `#` right at the top which supersedes the title.